nimsuggest/tests/massertish.nim

```diff
@@ -0,0 +1,5 @@
+# Template for testing defs
+
+template doAssertish*(cond: untyped, msg: string = "") =
+  ## template to allow def lookup testing
+  if not cond: quit()
```

Comment:
fakeAssert?
or at least:
`if not cond: quit(1)` (my understanding is you're not actually using this proc for anything other than definition lookup, but could be made a bit clearer)

Reply:
Done